### ISSUE TITLE
Extra Add Block screen on add group after edit block

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#1307]
An extra add block pop-up is shown to the user after clicking on a newly added group.

Steps to reproduce:
1. With Blocks in the other group
2. Right click on block select "Edit block"
3. Click OK (must be ok)
4. Click Groups tab
5. Add (group)
6. Click on name of new group
7. Popup for ad block appears

**Comment by KathrynBaker** [comment]
I have seen similar behaviours from the right-click, edit block method of accessing the config and then doing something else. It doesn't seem to matter if you click OK or cancel for me, or what you edit. e.g. as I wrote this I did:
1. Right click > edit block on a block in a named group
2. Used my Escape key
3. Went to the summary tab
4. Added a space to the end of my description and the configure block for my selected block pop up appears
